### PR TITLE
Renames, aliases, deep optics.

### PR DESCRIPTION
**Pull request by rpiaggio**

### CODE REVIEW

**Review by swalker2m — APPROVED** [review]
Looks good to me.

File: modules/core/shared/src/main/scala/lucuma/core/model/SpectralDefinition.scala [hunk 1]
```diff
@@ -25,20 +26,18 @@ sealed trait SpectralDefinition[T]
 object SpectralDefinition {
 
   final case class BandNormalized[T](
-    sed:          UnnormalizedSpectralEnergyDistribution,
```
Comment (swalker2m) [inline modules/core/shared/src/main/scala/lucuma/core/model/SpectralDefinition.scala:28]: My impression, probably wrong, is that SED is a pretty common (for astronomy) abbreviation but `USED` might seem odd.  If I were correct, we might consider `UnnormalizedSed` to shorten it, albeit not by as much?  At any rate, when it appears in the GraphQL API we might want to check whether `UnnormalizedSed` would be more recognizable to an astronomer.

Comment (rpiaggio) [reply modules/core/shared/src/main/scala/lucuma/core/model/SpectralDefinition.scala:28]: Sounds good, I'll just rename `UnnormalizedSpectralEnergyDistribution` to `UnnormalizedSED` then.

Comment (rpiaggio) [reply modules/core/shared/src/main/scala/lucuma/core/model/SpectralDefinition.scala:28]: Sounds good, I'll just rename `UnnormalizedSpectralEnergyDistribution` to `UnnormalizedSED` then.

**Pull request merged**